test/functional/fixtures/api/es-next/type/test.js

```diff
@@ -43,4 +44,20 @@ describe('[API] t.typeText()', function () {
                 expect(errs[0]).to.contains('> 19 |    await t.typeText(NaN, \'a\');');
             });
     });
+
+    if (!config.proxyless) {
+        it('Should not get selector the second time if the error was raised in the first.', function () {
+            return runTests('./testcafe-fixtures/type-test.js', 'Not found selector', {
+                shouldFail: true,
+                only:       'chrome',
+            })
+                .catch(function (errs) {
+                    expect(testReport.durationMs).lessThan(1100);
```

Comment:
Increase the `selectorTimeout` value to make test more stable (this test sometimes failed on my machine).